default.json

```diff
@@ -63,9 +63,6 @@
     },
     {
       "description": "Automatically merge minor and patch updates for Microsoft and trusted, well-known dependencies",

```

Comment:
We should remove all auto-merge setting in the default one since we move it to the others files.

Reply:
We could, people who are already using the default config would loose the feature tho